src/main/java/com/example/petmanagment/PetService.java

```diff
@@ -44,5 +44,12 @@ public Pet update(int id, BigDecimal weight) {
         existingPet.setOldWeight(oldWeight); // 前の体重をセット
         return existingPet;
     }
+
+    public Pet delete(int id) {
```

Comment:
voidでもいいかと思いますがいかがですか？
```suggestion
    public void delete(int id) {
```

Reply:
そうですね...voidにした方が良いかなと悩んだのですが、登録していた子の名前をレスポンスボディに返したかった（「この子のデータ消しちゃったよ」というメッセージも込めて…）ので、この実装にしましたｍｍ

Reply:
意図があるならOKです！